std/os.zig

```diff
@@ -2134,6 +2134,10 @@ pub const ArgIterator = struct {
     inner: InnerType,
 
     pub fn init() ArgIterator {
+        if (builtin.os == Os.wasi) {
+            @compileError("ArgIterator is not yet supported in WASI. Use argsAlloc and argsFree instead.");
```

Comment:
Would you mind putting "TODO" in here so that it gets noticed with #363?

Reply:
Done